_data/changelogs/component-checkbox.yml

```
@@ -2,6 +2,14 @@ title: Checkbox
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Updated radio and checkbox tile styling to have lighter borders.
```

Changing to `styles` could read better.

```suggestion
    summary: Updated radio and checkbox tile styles to have lighter borders.
```

---

```suggestion
    summary: Updated radio and checkbox tiles to have lighter borders, reducing visual noise.
```